packages/ui/src/components/molecules/Graph/LineGraph/index.tsx

```diff
@@ -20,13 +20,18 @@ export interface LineGraphProps {
     timestamp: number;
     value: number;
   }[];
-  formatYAxisTick?: (value: number | string) => string | number;
+  formatYAxisTick?: (
+    value: number | string,
+    isDiscreetMode?: boolean,
+  ) => string | number;
   formatTooltipValue?: (params: {
     timestamp: number;
     value: number;
+    isDiscreetMode?: boolean;
   }) => string[];
   formatTimestamp?: (timestamp: number) => string;
   color: string;
+  isDiscreetMode?: boolean;
```

Comment:
Can we move the `isDiscreetMode` variable to the callback functions instead of prop?